src/gateway/install/index.md

```diff
@@ -6,5 +6,8 @@ Kong can be installed on many different systems. From bare metal, to virtual mac
 
 This page outlines Kong's official support policy and provides links to detailed installation instructions for each.
 
+<i class="fa fa-check"></i>: Full production support
+<i class="fa fa-times"></i>: Local testing only
+
```

Comment:
```suggestion
Legend:
* <i class="fa fa-check"></i>: Full production support
* <i class="fa fa-times" style="color: #d44324"></i>: Local testing only

```
Hard to read as it is, in one line.
Added styling to X since it's only red by default in tables.